acl-lib/src/cmr/acl/acl_fetcher.clj

```diff
@@ -73,18 +73,38 @@
   [context]
   (:object-identity-types (cache/context->cache context acl-cache-key)))
 
+(defn- format-response
+  "Formats ACL search response by pulling ACLs out of items and converting map keys to kebab-case"
+  [response]
+  (map (comp :acl util/map-keys->kebab-case)
+       (get response :items)))
+
 (defn- process-search-for-acls
   "Calls acl search endpoint using converting object-identity-types
    and processes response and formats it for get-acls"
   [context object-identity-types]
-  (map (comp :acl util/map-keys->kebab-case)
-       (get
-        (access-control/search-for-acls (assoc context :token (config/echo-system-token))
-                                        {:identity-type (object-identity-types->identity-strings
-                                                         object-identity-types)
-                                         :include-full-acl true
-                                         :page-size 2000})
-        :items)))
+  (let [page-size 2000
```

Comment:
We should add a paging test (we can do this in a separate PR). For that test you might need to make the page-size configurable and change the value for the test.